web_widget_open_tab/static/src/js/widget.js

```diff
@@ -0,0 +1,82 @@
+odoo.define('web_widget_open_tab.FieldOpenTab', function(require) {
+    "use strict";
+
+    var AbstractField = require('web.AbstractField');
+    var field_registry = require('web.field_registry');
+    var ListRenderer = require('web.ListRenderer');
+    var core = require('web.core');
+    var config = require('web.config');
+    var qweb = core.qweb;
+    var _t = core._t;
+
+    var FieldOpenTab = AbstractField.extend({
+        description: "",
```

Comment:
any nice description instead?

Reply:
If description is shown, it will be written on the top of the tree, I think it is better to keep it blank

Reply:
a comment for your future self will help :wink:

Reply:
You are right :+1: